setup.py

```diff
@@ -1,62 +0,0 @@
-""" setup.py - Distutils setup file for PyBERT package
```

Comment:
Did you mean to delete this file?
Is it that `setup.py` becomes superfluous once you introduce a `pyproject.toml`?
Is that true for _all_ users, regardless of how out-of-date their own local Python infrastructure may be?
That is, are we _certain_ that no one needs the `setup.py` file anymore?

Reply:
That is a very good question, you are currently supporting python 3.8 in conda.  I'm testing 3.9/3.10 for pypi/setuptools support.  As long as their version of pip is within the past few years, it should know what a `pyproject.toml` file is and tell the user to update pip to use it.  The file itself specs the version of setuptools that it requires to install it.

Overall, it is meant to be one or the other not both.  I think you actually get some errors and warnings if you have both.

Reply:
Okay, given the magnitude of this update, as well as the fact that we're updating the major version number (alerting people to potential breakage of existing client code and/or build infrastructure if I understand versioning protocol correctly), let's leave `setup.py` behind and just go for it, fully committing to `pyproject.toml`.

Cross your fingers we don't get severely bitten! ;-)